server/api/controllers/message.controller.js

```diff
@@ -25,8 +25,15 @@ module.exports = function MessageController(gladys) {
       source_user_id: req.user.id,
       user: req.user,
       created_at: req.body.created_at || new Date(),
+      uuid: req.body.uuid,
```

Comment:
Be careful, this will be sent to the `message.create` function, that isn't supposed to accept a `uuid` attribute.

If you really need this id, you could generate the id here and use it when inserting the message in DB then, so ID are consistent.

Reply:
It's just used to communicate via websocket. No need to persist it. Fixed